payjoin-directory/src/lib.rs

```diff
@@ -138,9 +138,6 @@ fn init_ohttp() -> Result<ohttp::Server> {
 
     // create or read from file
     let server_config = ohttp::KeyConfig::new(KEY_ID, KEM, Vec::from(SYMMETRIC))?;
-    let encoded_config = server_config.encode()?;
-    let b64_config = BASE64_URL_SAFE_NO_PAD.encode(encoded_config);
-    info!("ohttp-keys server config base64 UrlSafe: {:?}", b64_config);
```

Comment:
fetch ohttp-keys by using GET `/ohttp-keys`

Reply:
It is in the bip 77 spec but not elsewhere in our coodebase.

We could call `info!` and print "fetch ohttp-keys by using GET `/ohttp-keys`" as an instruction for the operator.

Reply:
Cool, I forgot it was in the spec. I like that suggestion too.